.github/workflows/deploy-and-test.yaml

```
@@ -153,7 +153,7 @@ jobs:
     needs: [deploy]
     uses: ./.github/workflows/starknet-go-tests.yml
     with:
-      ref: 103d20c44ad4e07586a933e63564ae9505463959
+      ref: 495fe456fa62477096aa6f582600b068c75a598a
```

This is the latest commit from main. I've updated some things in some failing tests

```suggestion
      ref: b5d65c7aa351ef8a4bbb125f17dc7bc0fe37a5db
```